vite.config.ts

```diff
@@ -4,10 +4,36 @@ import tsconfigPaths from 'vite-tsconfig-paths'
 import nodePolyfills from 'rollup-plugin-polyfill-node';
 import { NodeGlobalsPolyfillPlugin } from '@esbuild-plugins/node-globals-polyfill'
 import svgr from 'vite-plugin-svgr' 
+import { htmlInjectionPlugin } from "vite-plugin-html-injection";
+const matomoSrcEvents = "./src/matomo/matomo_events.html";
+const matomoSrcLive = "./src/matomo/matomo_inject_live.html";
+const matomoSrcDev = "./src/matomo/matomo_inject_dev.html";
+const matomoSrc = process.env.VITE_NODE_URL === 'wss://mainnet-rpc.polymesh.network' ? matomoSrcLive : matomoSrcDev;
```

Comment:
this works but adding a dedicated env may be a better option and avoid a duplicate script. Otherwise all local forked instances may also trigger events to the `container_HgAzdcAg_dev_6f3f88f6ab703af55d52c94a.js` script and if we ever change our wss url it would break. e.g. someone uses wss://rpc.polymesh.network instead